pages/linux/qm-delsnapshot.md

```diff
@@ -0,0 +1,12 @@
+# qm delsnapshot
+
+> Delete a virtual machine snapshot.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Delete a snapshot:
+
+`qm delsnapshot {{vm_id}} {{snapshot_name}}`
+
+- Delete a snapshot from config (even if removing the disk snapshot fails):
```

Comment:
Perhaps this better because we don't abbreviations in example descriptions
```suggestion
- Delete a snapshot from a configuration file (even if removing the disk snapshot fails):
```